src/adaptations/device-layer/include/Weave/DeviceLayer/WeaveDeviceConfig.h

```diff
@@ -325,6 +325,16 @@
 #define WEAVE_DEVICE_CONFIG_ENABLE_WOBLE 1
 #endif
 
+/**
+ * WEAVE_DEVICE_CONFIG_SINGLE_WOBLE_CONNECTION
+ *
+ * Limit support for Weave-over-BLE (WoBLE) to a single connection.
+ * When set, WoBLE advertisements will stop while a WoBLE connection is active.
+ */
+#ifndef WEAVE_DEVICE_CONFIG_WOBLE_SINGLE_CONNECTION
+#define WEAVE_DEVICE_CONFIG_WOBLE_SINGLE_CONNECTION 0
```

Comment:
Can you open an issue to add support for this to the ESP32?

Reply:
Done: #400